src/blocks/app-and-main-menu/_fhi-footer.block.scss

```diff
@@ -0,0 +1,105 @@
+.fhi-footer {
+    margin-top: auto;
+    
+    &__container {
+        border-top: 1px solid $border-color;
+        margin-top: $fhi-space-3;
+        padding-top: $fhi-space-4;
+        padding-bottom: $fhi-space-4;
+        position: relative;
+    }
+}
+
+
+.fhi-footer--arch {
+    --fhi-footer-wave-left-position-start: -24%;
+    --fhi-footer-wave-left-position-end: -6%;
+    
+    overflow: hidden;
+    position: relative;
+
+    &::before,
+    &::after {
+        background-color: $fhi-red-1;
+        content: "";
+        position: absolute;
+    }
+    
+    &::before {
+        border-top-right-radius: 100%;
+        border-top-left-radius: 100%;
+        height: $fhi-px * 214;
+        left: var(--fhi-footer-wave-left-position-start);
+        top: $fhi-space-3;
+        width: 130%;
+    }
+
+    &::after {
+        bottom: 0;
+        height: 100%;
+        left: 0;
+        right: 0;
+        top: ($fhi-space-6 * 2);
+        transition: all .2s;
+    }
+
+    &__container {
```

Comment:
I rather you do `.fhi-footer__container` inside the modifier, and skip `fhi-footer--arch__container` in the markup